site/en/tutorials/audio/transfer_learning_audio.ipynb

```diff
@@ -131,27 +130,25 @@
       "source": [
         "## About YAMNet\n",
         "\n",
-        "YAMNet is an audio event classifier that takes audio waveform as input and makes independent predictions for each of 521 audio events from the [AudioSet](https://research.google.com/audioset/) ontology.\n",
```

Comment:
It's important not to loose the "independent predictions for each of 521 ..." here.

I talkes to the authors., The model really does do 512 independant binary classifications. Not a 512-way softmax. That distinction is important to correctly interpret the results.

Reply:
Done.

```
[YAMNet](https://github.com/tensorflow/models/tree/master/research/audioset/yamnet) 
is a pre-trained neural network that employs the [MobileNetV1](https://arxiv.org/abs/1704.04861)
depthwise-separable convolution architecture. It can use an audio waveform as input and make
independent predictions for each of the 521 audio events from the [AudioSet](http://g.co/audioset) corpus.
```